app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.split()[1] != command.split()[2]:
+        with (open(
+                f"{command.split()[1]}",
+                "r"
+        ) as first_file,
+            open(
+                f"{command.split()[2]}",
+                "w"
+        ) as copy_file):
```

Comment:
read this message please
https://mate.academy/chats/244801?selectedMessageId=3129519